apps/meeting/.eslintrc.json

```diff
@@ -0,0 +1,56 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react",
+        "@typescript-eslint"
+    ],
+    "rules": {
+        // Follow most common indentation of 2 spaces.
+        "indent": [
+            "error",
+            2
+        ],
+        "linebreak-style": [
+            "error",
+            "unix"
+        ],
+        "quotes": [
+            "error",
+            "single"
+        ],
+        "semi": [
+            "error",
+            "always"
+        ],
+        "comma-dangle": [
+            "error",
+            {
+                "arrays": "always",
+                "objects": "always",
+                "imports": "always",
+                "exports": "always"
+            }
+        ],
```

Comment:
I'm not sure about it, but adding trailing comma in the one-line statement looks weird to me.
```js
import { useNavigation, } from '../../providers/NavigationProvider';
```
Instead of "always" I would prefer "only-multiline". Do you think it's better to update it to below?
```
"comma-dangle": ["error", "only-multiline"],
```

Reply:
Used "always-multiline" over "only-multiline" to enforce the rule with "only-multiline" it is allowed and not required but with "always" it is required.

```
// "always-multiline"
var foo = {
    bar: "baz",
    qux: "quux", <- Required
};

// "only-multiline"
var foo = {
    bar: "baz",
    qux: "quux" <- Is ok to not add a comma here.
};
```

Source: https://eslint.org/docs/rules/comma-dangle